coral/src/app/main.tsx

```diff
@@ -2,11 +2,19 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import { RouterProvider } from "react-router-dom";
 import router from "src/app/router";
-
 import "@aivenio/design-system/dist/styles.css";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
+
+// @TODO add vite env
+const DEV_MODE = true;
+const queryClient = new QueryClient();
 
 ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
   <React.StrictMode>
-    <RouterProvider router={router} />
+    <QueryClientProvider client={queryClient}>
+      {DEV_MODE && <RouterProvider router={router} />}
```

Comment:
I think this is supposed to be:
```
<RouterProvider router={router} />
{DEV_MODE && <ReactQueryDevtools />}     
```

So router is always there, but dev tools only on dev

Reply:
🤦‍♀️ @ myself.